code/modules/shuttle/arrivals.dm

```diff
@@ -7,6 +7,7 @@
 	height = 15
 	dir = WEST
 	port_angle = 180
+	preferred_direction = WEST

```

Comment:
The parallax is always right. I don't know what you mean by saying its different?